cmd/system-probe/config/adjust_npm.go

```diff
@@ -77,4 +78,9 @@ func adjustNetwork(cfg config.Config) {
 		}
 		return nil
 	})
+
+	if common.DisablePESUnsupportedKernel(cfg.GetBool(evNS("network_process", "enabled"))) {
+		cfg.Set(evNS("network_process", "enabled"), false, model.SourceAgentRuntime)
+	}
+
```

Comment:
This is the wrong place in the code to be doing this. This file is purely doing feature config flag rewriting. Probing to see if particular features are supported is done at eBPF module startup.  Try to find the right place to do this (hint, it's somewhere in tracer.go)

Reply:
added it in tracer.go... lmk what you think

Reply:
This was put here so that other modules using this config also get affected by this change, not just the tracer.

Reply:
changed to original implementation after offline discussion